deps/wazuh_testing/wazuh_testing/modules/fim/utils.py

```diff
@@ -101,8 +103,6 @@ def create_registry(key, subkey, arch):
 
     if sys.platform == 'win32':
         try:
-            logger.info("Creating registry key " + str(os.path.join(fim.registry_class_name[key], subkey)))
-
```

Comment:
We are deleting it, because It is shown only if we run the tests with `-s` parameter or when tests fail. And it makes the logs unreadable in a lot of cases, because adds a lot of lines stating that the key's are being created (what has not happened yet. In any case, if we allow it to stay, I would change it so it is done after the key was created with no error.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4121/commits/78f1c5672e3be05e4f8b84a9812aacb96965b2a5